variables.tf

```diff
@@ -79,16 +79,6 @@ variable "worker_sg_ingress_from_port" {
   default     = "1025"
 }
 
-variable "kubeconfig_context_name" {
```

Comment:
What I've pitched above is probably flexible enough for what you're after @sdavids13 . Thoughts?

Reply:
With PR #49 I attempted to make the various kubeconfig options map entries but ran into issues with TF complaining that I was mixing and matching string/list/map value types in the same map. Also, do you know of a way to only override a single value instead of having to define everything?

Reply:
Now that we have a sensible value for context name, over riding it to something other than the cluster name seems pretty unlikely, no?